Controller/PostController.php

```diff
@@ -23,9 +23,11 @@
 class PostController extends Controller
 {
     /**
+     * @param Request $request
+     *
      * @return RedirectResponse
      */
-    public function homeAction()
+    public function homeAction(Request $request = null)
```

Comment:
I tried adding the request parameter to the homeAction in my own PostController:
```php

    public function homeAction(Request $request)
    {
        return $this->redirect($this->generateUrl('sonata_news_archive'));
    }
```
Got an exception:
```
Runtime Notice: Declaration of Core23\NewsBundle\Controller\PostController::homeAction() 
should be compatible with Sonata\NewsBundle\Controller\PostController::homeAction()
```